waku/v2/protocol/filter/filter_test.go

```diff
@@ -543,3 +543,67 @@ func (s *FilterTestSuite) BeforeTest(suiteName, testName string) {
 func (s *FilterTestSuite) AfterTest(suiteName, testName string) {
 	s.log.Info("Finished executing ", zap.String("testName", testName))
 }
+
+func (s *FilterTestSuite) TestStaticSharding() {
+	log := utils.Logger()
+	s.log = log
+	s.wg = &sync.WaitGroup{}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second) // Test can't exceed 10 seconds
+	s.ctx = ctx
+	s.ctxCancel = cancel
+
+	// Gen pubsub topic "/waku/2/rs/1024/0"
+	s.testTopic = protocol.NewStaticShardingPubsubTopic(uint16(1024), uint16(100)).String()
+
+	// Pubsub topics for neg. test cases
+	testTopics := []string{
+		"/waku/2/rs/100/1024",
+		"/waku/2/rs/1024/101",
```

Comment:
Again, as i had mentioned above...whatever we want can be achieved by using shards from same cluster. 

Let us not use shards from different cluster.

Reply:
Cluster IDs unified at https://github.com/waku-org/go-waku/pull/1060/commits/7404bb416fe513d2dd68387657767adaa4f5b852